src/modules/uORB/uORBDevices.cpp

```diff
@@ -1123,7 +1128,7 @@ void uORB::DeviceMaster::showTop(char **topic_filter, int num_filters)
 
 			if (ret > 0) {
 
-				ret = read(0, &c, 1);
+				ret = px4_read(STDIN_FILENO, &c, 1);
```

Comment:
`px4_read` cannot be used here, because on posix it uses the virtual FD's, but we're accessing STDIN_FILENO which is an OS FD.